app/models.py

```diff
@@ -300,6 +299,7 @@ class ApplicationActionType(StrEnum):
     MSME_CONFIRM_EMAIL = "MSME_CONFIRM_EMAIL"
     MSME_UPLOAD_ADDITIONAL_DOCUMENT_COMPLETED = "MSME_UPLOAD_ADDITIONAL_DOCUMENT_COMPLETED"
     MSME_RETRY_APPLICATION = "MSME_RETRY_APPLICATION"
+    MSME_ACCESSED_EXTERNAL_ONBOARDING = "MSME_ACCESSED_EXTERNAL_ONBOARDING"
```

Comment:
The others are just the infinitive, e..g. "FI_START_APPLICATION", etc.

```suggestion
    MSME_ACCESS_EXTERNAL_ONBOARDING = "MSME_ACCESS_EXTERNAL_ONBOARDING"
```